talks/README.md

```diff
@@ -2,3 +2,9 @@ The talks folder can contain any materials used for meetups at the Bay Area WTD
 
 This README is the place to store links to your recorded WTD events, if they are stored on YouTube. 
 
+**talks - Table of Contents** 
+
+| Date     | Loc | Topic                    | Filename <presenter_topic_name> | 
+|----------| ----|--------------------------|---------------------------------|
+| 2017/4/3 | SB  | Localizing Documentation | BeeH_<topic_name>.pdf           |
```

Comment:
I know that these are just sample entries, but it still seems confusing that we'd have .pdf as the file extension for a link to a video. If this table is intended for multiple formats (presentation slides in PDF, presentations in PPTX format, html links to YouTube videos, actual video files), then either add a sample link for each type or change the lead-in text for the table.